src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -145,6 +162,69 @@ public static SafeFreeCredentials AcquireCredentialsHandle(X509Certificate? cert
             return AcquireCredentialsHandle(direction, secureCredential);
         }
 
+        public static unsafe SafeFreeCredentials AcquireCredentialsHandleNew(X509Certificate? certificate, SslProtocols protocols, EncryptionPolicy policy, bool isServer)
+        {
+            int protocolFlags = GetProtocolFlagsFromSslProtocols(protocols, isServer);
+            Interop.SspiCli.SCH_CREDENTIALS.Flags flags;
+            Interop.SspiCli.CredentialUse direction;
+
+            if (!isServer)
```

Comment:
Nit: why not flip the polarity to avoid the negative, e.g.
```C#
if (isServer)
{
   ...
}
else
{
    ...
}
```

Reply:
updated.